src/main/java/org/betonquest/betonquest/objectives/MobKillObjective.java

```diff
@@ -27,13 +29,13 @@
 @CustomLog
 public class MobKillObjective extends CountingObjective implements Listener {
 
-    protected EntityType mobType;
+    private final Set<String> entities = new HashSet<>();
     protected String name;
     protected String marked;
 
     public MobKillObjective(final Instruction instruction) throws InstructionParseException {
         super(instruction, "mobs_to_kill");
-        mobType = instruction.getEnum(EntityType.class);
+        Collections.addAll(entities, instruction.getArray());
```

Comment:
The event will no longer check if the entities are a valid mob type when parsing the event.

Rather use `instruction.getList(mob -> instruction.getEnum(mob, EntityType.class));` to get a list of entity types